drivers/espi/espi_mchp_xec_v2.c

```diff
@@ -1169,27 +1162,81 @@ static void vw_pltrst_handler(int girq_id, int src, void *user)
 	}
 }
 
+static void vw_sus_stat_handler(int girq_id, int src, void *user)
+{
+	const struct device *dev = (const struct device *)user;
+
+	notify_system_state(dev, ESPI_VWIRE_SIGNAL_SUS_STAT);
```

Comment:
@jvasanth1 none of these VW reflect a system state so these should "notify_host_warning"

Reply:
ok will update to invoke `notify_host_warning` instead; 
do we need to update the function `notify_host_warning` to handle `ESPI_VWIRE_SIGNAL_SUS_STAT`

Reply:
If you mean driver automatic handling acknowledge, no it's not required.

Reply:
Thanks,
updated the code